### DISCUSSION TITLE
Request for Access to Dictionary in PaddleOCR English Language Pretrained Model for Fine-Tuning

### DISCUSSION THREAD

**Discussion by IbrarBabar009** [PaddlePaddle/PaddleOCR#12989] [Q&A]
I am using the PaddleOCR pretrained model for the English language [link](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_en/models_list_en.md#22-english-recognition-model)

The dictionary path in the English YAML **en_dict.txt** file contains only 62 characters. 

`0
1
2
3
4
5
6
7
8
9
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
A
B
C
D
E
F
G
H
I
J
K
L
M
N
O
P
Q
R
S
T
U
V
W
X
Y
Z`

I tested an image that contains ">>>" and other special characters, and the model correctly identified it. This implies that the actual dictionary must include special characters as well.

I need the actual dictionary to fine-tune this model.

Is there any way I can view the dictionary of the PaddleOCR English language pretrained model? I would like to fine-tune it.

**Comment by IbrarBabar009** [comment]
@WenmuZhou @dyning @LDOUBLEV @tink2123 @MissPenguin @Topdu @Evezerest @littletomatodonkey @andyjpaddle @weisy11 @D-DanielYang @Topdu @sdcb @ZeyuChen @bingooo please help?????

**Comment by kirill-push** [comment]
Hi @IbrarBabar009 
Did you find the actual dictionary?
